samples/contrib/versioned-pipeline-ci-samples/kaggle-ci-sample/submit_result/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.7
```

Comment:
Similar to another Dockerfile. Can we base on google/cloud-sdk?

Reply:
Thanks for the advice. Will modify.